public/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,75 @@
+
+### Spamming signers {#spamming-signers}
+
+Another small attack vector is malicious signers injecting new vote proposals inside every block they mint. Since nodes need to tally up all votes to create the actual list of authorized signers, they need to track all votes through time. Without placing a limit on the vote window, this could grow slowly, yet unbounded. The solution is to place a _moving_ window of W blocks after which votes are considered stale. _A reasonable window might be 1-2 epochs._
+
+### Concurrent blocks {#concurrent-blocks}
+
+In a Proof-of-Authority (PoA) network, if for example there are 10 authorized signers and each signer is allowed to create 1 block out of 20, then at any given time, 11 miners can create blocks. To prevent them from racing to create blocks, each signer adds a small random "offset" to the time they release a new block. This reduces the occurrence of small forks but still allows occasional forks, as seen on the Ethereum Mainnet. If a signer misuses their authority and causes disruptions, they can be voted out of the network.
+
+## Pros and cons {#pros-and-cons}
+
+| Pros                                                                                                                                                      | Cons                                                                                                                                                  |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Scalable more than other popular mechanisms such POS and POW, as it's based on a limited number of block signers                                          | PoA networks typically have a relatively small number of validating nodes. This makes a PoA network more centralized.                                 |
+| PoA blockchains are incredibly cheap to run and maintain                                                                                                  | becoming an authorized signer is typically out of reach for an ordinary person, because the blockchain requires entities with established reputation. |
+| The transactions are confirmed very quick as it could reach less than 1 second because only limited number of signers are required to validate new blocks | malicious signers could reorg,double spend,censor transactions in the network, those attacks are mitigated but still possible                         |
```

Comment:
```suggestion
| The transactions are confirmed very quick as it could reach less than 1 second because only limited number of signers are required to validate new blocks | malicious signers could reorg, double spend, censor transactions in the network, those attacks are mitigated but still possible                         |
```